Userland/Utilities/ping.cpp

```diff
@@ -180,6 +180,15 @@ int main(int argc, char** argv)
                 return 1;
             }
 
+            char ihl = -1;
+            memcpy(&ihl, pong_packet.data(), sizeof(char));
```

Comment:
Since we are extracting a single value we can probably just use:
`auto ihl_with_better_name = *pong_packet.data(); & 0x0F;`